docs/variant-specific/pink.mdx

```diff
@@ -253,24 +256,67 @@ These conventions apply to any variant with a pink (touched by all ranks) suit.
+  - Cathy discards.
+  - Donald knows that normally, clues do not cause non-left-most cards to become _Prompted_. Thus, Donald knows that Bob must be responding to Alice's _Bubblegum Prompt_ on a card that is not actually a 3.
+  - The next card after a pink 3 is a pink 4, so Donald knows that the card that Alice focused is a pink 4. Subsequently, since Alice's clue was a _Play Clue_, the pink 4 must be playable. In order for a pink 4 to be playable, someone has to have a pink 3. Donald does not see it anywhere else, so he determines that he must have the pink 3 on slot 3 (since that is the only other clued card that could possibly be a pink 3).
+- It is possible for a clue to be both a _Bubblegum Finesse_ and a _Bubblegum Prompt_ at the same time.
+  - In the previous example, we can modify Donald's hand such that he only holds the pink 4 (instead of holding both the pink 4 and the pink 3).
+  - In the previous example, Cathy discarded, but in the modified example, Cathy would know that the pink 4 was promised to be playable. And Cathy would not see anyone else holding the pink 3, so Cathy would have to respond by playing a pink 3 from her _Finesse Position_.
 
-### The Bubblegum Bluff
+### The Bubblegum Bluff (Standard Form)
 
 - Similar to a _Bubblegum Finesse_, it is also possible to perform a _Bubblegum Bluff_.
 - For example, in a 3-player game:
   - It is the first turn and nothing is played on the stacks.
   - Alice clues number 3 to Cathy, touching a pink 2 on slot 5.
   - Bob knows that Alice is violating _Pink Promise_. This must mean that he is supposed to play his slot 3 card.
   - Bob blind-plays slot 3 and it is a red 1.
-  - Cathy knows that if this was a _3 Bluff_, then Bob would have played his _Finesse Position_. So it cannot be a _3 Bluff_.
+  - Cathy knows that if this were a _3 Bluff_, then Bob would have played his _Finesse Position_. So it cannot be a _3 Bluff_.
   - Thus, Cathy knows that this must be a _Bubblegum_ clue. Since red 1 is not a pink card and Alice was in _Bluff Seat_ for Bob, this must be a _Bubblegum Bluff_.
   - Cathy marks her slot 5 card as the one-away-from-playable pink card, the pink 2.
 
 <BubblegumBluff />
 
-- _Bubblegum Bluffs_ can only be performed on non-pink cards; otherwise, it will look like a _Double Finesse_ with one part _Bubblegum Finesse_ and one part normal _Finesse_.
+- _Bubblegum Bluffs_ can only be performed on non-pink cards; otherwise, it will look like a _Bubblegum Finesse_.
 - Note that it is impossible to _3 Bluff_ and _Bubblegum Bluff_ at the same time.
+- Note that whenever a pink card plays in response to a _Bubblegum_ move, it is guaranteed to the team to be a _Finesse_. (In other words, you cannot use a _Bubblegum Bluff_ with a target of a pink 3 to bluff out a pink 1.)
+
+### The Bubblegum Bluff (Two-Away-From-Playable Form)
+
+- Normally, _Bubblegum Bluffs_ are always performed by touching _one-way-from-playable_ pink cards.
```

Comment:
```suggestion
- Note that, as with a standard _Bluff_, the receiver of a _[Bubblegum Bluff](#the-bubblegum-bluff)_ marks their pink card as the lowest-possible, useful value it could be (given the team's preceding actions).
```

Reply:
(oops, dupe suggestion? internet lag I suppose)